Marlin/src/module/stepper.cpp

```diff
@@ -1508,14 +1508,14 @@ void Stepper::isr() {
     #if ENABLED(FT_MOTION)
 
       if (using_ftMotion) {
-        if (!nextMainISR) {
-          nextMainISR = FTM_MIN_TICKS;
-          ftMotion_stepper();
-          endstops.update();
-          TERN_(BABYSTEPPING, if (babystep.has_steps()) babystepping_isr());
+        if (!nextMainISR) {               // Main ISR is ready to fire during this iteration?
+          nextMainISR = FTM_MIN_TICKS;    // Set to minimum interval (a limit on the top speed)
+          ftMotion_stepper();             // Run FTM Stepping
+          IF_DISABLED(ENDSTOP_INTERRUPTS_FEATURE, endstops.update());         // Check endstops now
```

Comment:
I'm presuming that this call isn't needed when endstops are being updated by their own interrupt.

For consistency with the ESP32 I2C handling, the calls to both `endstops.update()` and `babystepping_isr()` could be moved to the end of `ftMotion_stepper`. Or a small `inline` function that calls all three could be created and added to `stepper.h`.